tests/meta/clean/test_clean_flow_cells_api.py

```diff
@@ -0,0 +1,246 @@
+"""Tests for the clean flow cells API."""
+import time
+from typing import List
+
+import mock
+import pytest
+from housekeeper.store.models import File
+
+from cg.constants import SequencingFileTag
+from cg.constants.time import TWENTY_ONE_DAYS_IN_SECONDS
+from cg.exc import CleanFlowCellFailedError, HousekeeperBundleVersionMissingError
+from cg.meta.clean.clean_flow_cells import CleanFlowCellAPI
+from cg.store import Store
+from cg.store.models import Flowcell, SampleLaneSequencingMetrics
+
+
+def test_get_flow_cell_from_statusdb(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to get a flow cell from statusdb."""
+
+    # GIVEN a clean flow cell api with a store that contains a flow cell
+
+    # WHEN retrieving the flow cell from the store
+    flow_cell: Flowcell = flow_cell_clean_api_can_be_removed.get_flow_cell_from_status_db()
+
+    # THEN a flow cell is retrieved
+    assert isinstance(flow_cell, Flowcell)
+
+
+def test_is_flow_cell_in_statusdb(flow_cell_clean_api_can_be_removed: CleanFlowCellAPI):
+    """Test to check that a flow cell is in statusdb."""
+    # GIVEN a clean flow cell api with a store that contains a flow cell
+
+    # THEN checking whether a flow cell is in statusdb is TRUE
```

Comment:
I would say it is important to add a when statement for readability, it makes the reading of tests much smoother
```suggestion
    # WHEN checking if the flow cell is in StatusDB
    is_in_statusdb: bool = flow_cell_clean_api_can_be_removed.is_flow_cell_in_statusdb()

    # THEN the flow cell is in status db
    assert is_in_statusdb
```